test/functional/p2p_filter.py

```diff
@@ -27,12 +29,20 @@
     p2p_lock,
 )
 from test_framework.script import MAX_SCRIPT_ELEMENT_SIZE
+from test_framework.script_util import key_to_p2wpkh_script
 from test_framework.test_framework import BitcoinTestFramework
+from test_framework.wallet import MiniWallet
+
+
+def random_p2wpkh():
+    key = ECKey()
+    key.generate()
+    return key_to_p2wpkh_script(key.get_pubkey().get_bytes())
```

Comment:
wouldn't it be better to add this function in miniwallet? then if required it can be used in other tests as well.

Reply:
Good idea, done. Note though that I didn't put it directly into the MiniWallet class (since it doesn't need any of its internal state), but as global function into the test_framework/wallet.py module.